### PR TITLE
Add Nurse-1975

### PR DESCRIPTION
**Pull request by chrzyki**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Concept list note in `conceptlists.tsv` is WIP. More info needed here @LinguList?

### CODE REVIEW

**Comment by chrzyki** [comment]
Also includes two small typo fixes for list notes.

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -373,16 +373,17 @@ Carvalho-2021-206	de Carvalho, Fernando O.	2021		206	basic, areal	English	Iñapa
...
-Goddard-2002-59	Goddard, Cliff and Wierzbicka, Anna	2002		59	specific	English	Global		Goddard2002		This list contains concept labels for 59 semantic primes sorted into different categoried based on [Natural Semantic Metalanguage](https://de.wikipedia.org/wiki/Natural_Semantic_Metalanguage)) approach. The authors claim the concepts to be universal.	14	
+Goddard-2002-59	Goddard, Cliff and Wierzbicka, Anna	2002		59	specific	English	Global		Goddard2002		This list contains concept labels for 59 semantic primes sorted into different categorized based on [Natural Semantic Metalanguage](https://de.wikipedia.org/wiki/Natural_Semantic_Metalanguage)) approach. The authors claim the concepts to be universal.	14
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists.tsv:386]: Thanks for noticing this typo! But I intended to write "categorie**s**" instead of "categorized" :)

Comment (chrzyki) [reply concepticondata/conceptlists.tsv:386]: Whops, sorry! Will fix momentarily.

**Comment by LinguList** [comment]
Please @chrzyki, let us close this PR now. And let us proceed as follows: based on the lexibank-glosses in the list I sent to you, you map them with the actual `concepts.csv` which have concepticonmappings in `etc/concepts.csv` in the tls directory. Once these are merged, you can update a proper Nurse-1975-1038 list to concepticon, and we make a quick not so thorough review for now, okay?

**Comment by chrzyki** [comment]
Yep, sounds good.

**Comment by LinguList** [comment]
Comparing data with comparalex: 
http://comparalex.org/index.php?page=stdlist&id=16, it woudl be good to 
mention in the NOTE that we deliberately exclude the concept with 
numbers like 100a, etc., as they are not reliable for us.

**Pull request closed without merging**